### DISCUSSION TITLE
Welcome to Static-website Discussions!

### DISCUSSION THREAD

**Discussion by thuongtruong1009** [thuongtruong1009/portfolio#1] [Q&A] [answered]
## 👋 Welcome to github.com/thuongtruong1009!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BEPb** [comment, marked as answer]
Thanks for this repository.

**Reply by SayanthRock** [reply]
OMG

**Comment by Madkunt** [comment]
Hello,
This is a great idea @BEPb,
Thanks for implementing this repository @thuongtruong1009.

**Reply by Geek-Cheth** [reply]
Yeah thank you

**Comment by aruncs31s** [comment]
Can i use this for my own purpose , i kinda need it for making and to play with html and css because it looks neat and beautyful when i open it on my firefox web browser , i know you worked hard for this that's why im asking , and idont want to get a feeling like im steeling something  and @thuongtruong1009 thanks for this

**Reply by thuongtruong1009** [reply]
> Can i use this for my own purpose , i kinda need it for making and to play with html and css because it looks neat and beautyful when i open it on my firefox web browser , i know you worked hard for this that's why im asking , and idont want to get a feeling like im steeling something and @thuongtruong1009 thanks for this

Oh, yes of course bro!
Thank you for your interest!

**Reply by aruncs31s** [reply]
Oh great Thank you @thuongtruong1009

**Comment by tenmark86** [comment]
Requesting for permission to implement this and use it with your approval

**Reply by thuongtruong1009** [reply]
@tenmark86 
Thank you for your interest. If you have any problems, you can open a [new issue](https://github.com/thuongtruong1009/portfolio/issues/new/choose) or open a new PR 
Thank you very much 🥰

**Comment by Mkiige** [comment]
This is great,thank you.

**Comment by nandini-gangrade** [comment]
Thanks for this repo:-D

**Comment by jirafey** [comment]
this is great!

**Comment by LOVKUSH9888** [comment]
Great community ⭐

**Comment by tsaol** [comment]
It is a grate repository.

**Comment by 1AoB** [comment]
Thanks for this repository.

**Comment by rubikproxy** [comment]
Thanks for this repository

**Comment by Geek-Cheth** [comment]
Its really a nice one.

**Comment by nguyenhoanghai1605** [comment]
Hello,
This is a great repository ⭐⭐⭐
@thuongtruong1009 thanks for this.

**Comment by Alxdelira** [comment]
Thank you for this repository, it's brilliant!

**Comment by raviwijerathna1** [comment]
Thanks for this repository.

**Comment by thuongtruong109** [comment]
I have migrated to a new account at https://github.com/thuongtruong109. Hope to reconnect with all

**Comment by mmaachado** [comment]
that's a very useful repository, thanks!
